package.json

```diff
@@ -3,7 +3,14 @@
   "version": "10.1.1",
   "type": "commonjs",
   "description": "Turns off all rules that are unnecessary or might conflict with Prettier.",
-  "repository": "prettier/eslint-config-prettier",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/prettier/eslint-config-prettier.git"
+  },
+  "homepage": "https://github.com/prettier/eslint-config-prettier#readme",
+  "bugs": {
+    "url": "https://github.com/prettier/eslint-config-prettier/issues"
+  },
```

Comment:
```suggestion
  "repository": "git+https://github.com/prettier/eslint-config-prettier.git"
```

Reply:
<img width="476" alt="image" src="https://github.com/user-attachments/assets/11b6ea61-c9be-479c-adbb-61d0f59f20bc" />

`repository` and `homepage` is alreay fine on npm?

Reply:
And I didn't see any issue when run `npm bugs eslint-config-prettier`.

Reply:
Hi @JounQin, when downloading packages from the normal npm registry it's fine and the url will show when hovering over the package in `package.json` without the need for a `"homepage"` property:
![image](https://github.com/user-attachments/assets/07bf980f-e5c6-4dc6-85d0-08ce2911f576)

It's when attempting to hover over the package when using a private npm registry, such as in Azure DevOps, is when the homepage url is not shown.

Reply:
@Fdawgs Does only change to `"repository": "git+https://github.com/prettier/eslint-config-prettier.git"` help?

Reply:
@JounQin Unfortunately not. It's just the homepage that needs adding.

The bugs and repository changes were just extras I thought I'd add whilst I was looking at the package.json, so can be discarded.

Reply:
<img width="824" alt="image" src="https://github.com/user-attachments/assets/057336e5-4bfa-42c0-8e52-33039595b9a4" />

Does this work for you? Or `repository` change can also be discarded?